packages/shared/src/utils/location/location.ts

```diff
@@ -1,4 +1,16 @@
-export const getLocation = (location_list, value, type) => {
+type TLocationList = {
+    identity: {
+        services: {
+            idv: object;
+            onfido: object;
+        };
+    };
+    phone_idd: string;
+    text: string;
+    value: string;
+};
```

Comment:
just a suggestion:

```
type TType = {
    text: string;
    value: string;
}

type TLocationList = TType & {
    identity: {
        services: {
            idv: object;
            onfido: object;
        };
    };
    phone_idd: string;
};

```

and then : 

`export const getLocation = (location_list: TLocationList[], value: string, type: keyof TType ) `